cmd/podman/system/connection/add.go

```diff
@@ -75,15 +75,14 @@ func init() {
 	flags.BoolVarP(&cOpts.Default, "default", "d", false, "Set connection to be default")
 }
 
-func add(cmd *cobra.Command, args []string) error {
+func Add(cmd *cobra.Command, args []string) error {
```

Comment:
is this necessary?

Reply:
Sorry, this is a mistake thanks for catching